README.md

```diff
@@ -55,7 +55,7 @@ join Backdrop's developer community by
 
 When submitting a pull request, please make sure there is an open issue in the
 [issue tracker](https://github.com/backdrop/backdrop-issues/issues) and
-reference it in the description on the pull request.
```

Comment:
I don't see a problem with this original wording.

Reply:
@BWPanda i think "of pull request" sounds better than "on pull request" in English

Reply:
If you say I would revert this change to orginial wording

Reply:
I think the original is more accurate - there is a "description" field on github's PR screen.